custom/intrahealth/sqldata.py

```diff
@@ -773,16 +773,17 @@ class SumAndAvgQueryMeta(IntraHealthQueryMeta):
 
     def _build_query(self, filter_values):
         key_column = sqlalchemy.column(self.key)
+        group_by_columns = [sqlalchemy.column(c) for c in self.group_by]
```

Comment:
It might have been a bit cleaner to override `group_by`

```
@property
def group_by(self):
    return [sqlalchemy.column(c) for c in super(XXX, self).group_by]
```

Reply:
I had to chose this, since `group_by` itself seems to be used in other clauses where just text is sufficient.